platform/contact-center/training-video-process.md

```diff
@@ -1,9 +1,17 @@
-### Contact Center Training Video Process
+# Contact Center Training Video Process
```

Comment:
I'd rather we call this "Contact Center Product Demo Video Process"

Reply:
Sounds good.  Updated.